src/app/portlets/dot-edit-content/services/dot-edit-content-html.service.ts

```diff
@@ -7,13 +7,18 @@ import { DotDragDropAPIHtmlService } from './html/dot-drag-drop-api-html.service
 import { GOOGLE_FONTS } from './html/iframe-edit-mode.js';
 import { DotEditContentToolbarHtmlService } from './html/dot-edit-content-toolbar-html.service';
 import { DotDOMHtmlUtilService } from './html/dot-dom-html-util.service';
+import { MODEL_VAR_NAME } from './html/iframe-edit-mode.js';
+import { Subject } from 'rxjs';
 
 @Injectable()
 export class DotEditContentHtmlService {
     contentletEvents: BehaviorSubject<any> = new BehaviorSubject({});
     iframe: ElementRef;
 
-    private addContentContainer: string;
+    modelChange: Subject<any> = new Subject();
```

Comment:
I think ```pageModel``` it's better name for this, modelChange sounds like an event and in reality what your are doing is subscribing to the page model.